startup.py

```diff
@@ -0,0 +1,373 @@
+# Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
Ah, you're right. Seems like the macOS review app doesn't export the right resolution when trying to save the icon. I had to click on the right icon and drag it out from the app and into the folder to get the right resolution.

#badUX